region/maridia/inner-yellow/Maridia Elevator Room.json

```diff
@@ -953,6 +986,36 @@
       "flashSuitChecked": true,
       "note": "Shinespark vertically up the right side of the shaft in order to avoid getting hit by the Ripper."
     },
+    {
+      "link": [2, 3],
+      "name": "Tricky Grapple Jump",
+      "entranceCondition": {
+        "comeInWithGrappleSwing": {
+          "blocks": [
+            {"position": [7, 3], "note": "The Precious Room"},
+            {"position": [7, 2], "note": "The Moat"}
+          ]
+        }
+      },
+      "requires": [
+        "canTrickyGrappleJump"
```

Comment:
These two feel easier than other tricky's.
Pant's start was still hard.

Reply:
Agreed, moved those two down to `canPreciseGrappleJump` now.